pkg/proxy/proxy.go

```diff
@@ -28,13 +28,15 @@ type KindType string
 const (
 	AlertManagerKind  KindType = "alertmanager"
 	ThanosQuerierKind KindType = "thanos-querier"
+	PersesKind        KindType = "perses"
 )
 
 type ProxyPort int
 
 const (
 	AlertmanagerPort  ProxyPort = 9444
 	ThanosQuerierPort ProxyPort = 9445
+	PersesPort        ProxyPort = 9446
```

Comment:
Make sure to update the port argument info in plugin-backend to include this port as reserved

Reply:
I'm going to remove the proxy so the CR can handle it.